src/main.rs

```diff
@@ -239,29 +229,19 @@ fn migrate(data_string: String) -> Result<String, anyhow::Error> {
         );
     }
 
-    //  /tmp/$tmpdir/NM-migrated, ich weiß nicht, ob das der richtige Path ist
     let migrated_file_location =
-        format!("{}/NM-migrated", tmp_dir.path().display().to_string()).to_string();
+        format!("{}/NM-migrated", tmp_dir.path().display());
 
     Command::new("tar")
         .args([
             "cf",
             output_path_str,
-            //&format!("/migration-tmpdir/NM-migrated/"),
             "-C",
-            // tmp_dir.path().to_str().unwrap(),
             &migrated_file_location,
-            // input_path_filename,
             ".",
         ])
         .output()?;
-    let ls_cmd = Command::new("ls").args([&migrated_file_location]).output()?;
-    println!("ls: {}", String::from_utf8_lossy(&ls_cmd.stdout));
-
-
-    println!("output_path_str: {}", output_path_str);
     Err(anyhow::anyhow!("testing"))
```

Comment:
```suggestion
    Ok(output_path_str.to_string())
```
You've deleted the wrong line :slightly_smiling_face: